src/common/locales/en/translation.json

```diff
@@ -22,7 +25,30 @@
     "stop": "Stop",
     "stopButton": "Stop button"
   },
+  "inputs": {
+    "directory": {
+      "openInFileExplorer": "Open in File Explorer",
+      "selectDirectory": "Select a directory..."
+    },
+    "file": {
+      "copyFileName": "Copy File Name",
+      "copyFullFilePath": "Copy Full File Path",
+      "openInFileExplorer": "Open in File Explorer",
+      "selectFile": "Select a file..."
+    }
+  },
   "loading": "Loading...",
+  "outputs": {
+    "largeImage": {
+      "imageNotAvailable": "Image not available."
+    },
+    "ncnn": {
+      "modelNotAvailable": "Model data not available."
+    },
+    "pytorch": {
+      "modelNotAvailable": "Model data not available."
+    }
```

Comment:
I wonder, should those 2 (and ONNX when added) be unified?

Reply:
Probably